### DISCUSSION TITLE
What is V2? And how to update to it?

### DISCUSSION THREAD

**Discussion by pratyushgguptaa** [JanDeDobbeleer/oh-my-posh#1692] [Troubleshoot] [answered]
made a theme
wanted to create a pull request
saw other pull requests for new theme
@JanDeDobbeleer mentioned to update to v2 and use the --migrate option for config version 1.
So what is it? can someone point me in the right direction.

current oh-my-posh version:
7.5.1

**Comment by JanDeDobbeleer** [comment, marked as answer]
The current version is v1, there was no version before 😁 Just update to the latest version of oh-my-posh and try the migration.
